kvdb-rocksdb/src/lib.rs

```diff
@@ -261,6 +261,10 @@ impl DBAndColumns {
 			}
 		}
 	}
+
+	fn maybe_property(&self, col: usize, prop: &str) -> Option<usize> {
```

Comment:
Just a personal opinion but I think I'd prefer to keep this logic in the original place; seems like overkill to have its own method.

Reply:
what original place?

Reply:
Sorry, I meant: this method is called only from one place, so this one-liner doesn't buy us much in terms of cleaner code.